src/views/AmalgamationRegular/ReviewConfirm.vue

```diff
@@ -131,16 +131,28 @@
       <header>
         <h2>Folio or Reference Number for this Filing</h2>
         <p class="mt-4">
-          [*** TODO: blurb ***]
+          Enter the folio or reference number you want to use for this filing for you own tracking purposes. The
+          Business Folio or Reference Number is displayed below (if available). Entering a different value below will
+          not change the Business Folio or Reference Number. Only the number below will appear on the transaction report
+          and receipt for this filing.
         </p>
       </header>
 
       <v-card
         flat
         class="mt-6"
       >
-        <div class="pa-4">
-          [*** TODO: Folio Number component ***]
+        <div
+          class="px-4 pt-8 pb-6"
```

Comment:
Try to not use bottom padding, and instead use top padding on the element below this one.

Rationale: An element could add padding below itself to provide whitespace to the next element. Or, an element could add padding above itself to provide whitespace to the previous element. It doesn't matter which approach you take as long as you're consistent (within the same file), otherwise it gets really confusing/complicated.

Reply:
I applied bottom margin as space on top and bottom is different in the component. I have changed it to `class="px-4 py-8"` which solves the issue of uneven top and bottom space. .

with `class="pa-4"`

![image](https://github.com/bcgov/business-create-ui/assets/116035339/5490f5f6-4fd2-4596-b605-3c7b41db1ed3)

with `class="px-4 py-8"`

![image](https://github.com/bcgov/business-create-ui/assets/116035339/d6f7ce9f-bbba-4c65-bb04-9316a24bdfe8)

Reply:
Second one looks good 👍